src/routes/api/index.js

```diff
@@ -15,5 +15,8 @@ router.use('/api/v1/auth', userRoutes);
 // set up auth routes v1
 router.use('/api/v1/auth', userRoutes);
 
+// Assign user roles
+router.use('/api/v1/', userRoutes);
```

Comment:
isn't this a duplication
and from my point of view, this would mean that 
`localhost:300/api/v1/auth/facebook` and `localhost:300/api/v1/facebook`
can be accessed using both routes way

please if possible use just `/api/v1/auth` or `/api/v1` if you can

Reply:
@blaise82 it is not, because that route `/api/v1/auth` was used for authentication but now the feature am working on is not auth, so I have `/api/v1/` check users routes you will see the difference

Reply:
@AbdoulNuru great